nle/nethack/actions.py

```diff
@@ -76,6 +89,12 @@ class MiscAction(enum.IntEnum):
     MORE = ord("\r")  # read the next message
 
 
+class UnsafeActions(enum.IntEnum):
+    # currently these result in an error or undesirable behaviour
+    HELP = ord("?")  # give a help message
+    PREVMSG = C("p")  # view recent game messages
```

Comment:
So, is 'p' not in the action list?
Can this not be handled like quit, save, etc. below?

Reply:
No - control+P isnt. Luckily we dont need this action at all, so thats fine.